src/provisioner/cluster.py

```diff
@@ -75,6 +77,12 @@ def add_node(self, node: dict):
         self._validate(node)
         self._node_list.append(node)
 
+    @staticmethod
+    def get_machine_id(machine_id: str):
+        result = hashlib.md5(machine_id.encode())
+        node_id= result.hexdigest()
+        return node_id
```

Comment:
Return should be machine_id as the function name suggests.

Reply:
done